subscribers/tests/test_forms.py

```diff
@@ -1,12 +1,12 @@
 from http import HTTPStatus
 
 from django.contrib.messages import get_messages
-from django.test import SimpleTestCase
+from django.test import TransactionTestCase
 
 from subscribers.forms import SubscriberForm
 
 
-class TestSubscriberForm(SimpleTestCase):
+class TestSubscriberForm(TransactionTestCase):
```

Comment:
Instead of TransactionTestCase, we can use just TestCase.

Reply:
No we cant because the test fails because of that.
```
AssertionError: Database queries to 'default' are not allowed in SimpleTestCase subclasses. Either subclass TestCase or TransactionTestCase to ensure proper test isolation or add 'default' to subscribers.tests.test_forms.TestSubscriberForm.databases to silence this failure.
```

Reply:
Yes not `SimpleTestCase`. That does not work with Db transtaction. Just `TestCase`.  https://docs.djangoproject.com/en/3.2/topics/testing/tools/#provided-test-case-classes

Reply:
Okay alright